testchain/node/src/chain_spec.rs

```diff
@@ -240,6 +279,12 @@ fn testnet_genesis(
 		security: SecurityConfig {
 			initial_status: if start_shutdown { StatusCode::Shutdown } else { StatusCode::Error },
 		},
+		stellar_relay: StellarRelayConfig {
+			validators,
+			organizations,
+			is_public_network: true,
```

Comment:
`is_public_network` means that it's connected to mainnet, right? I don't get why, even in the testnet config, `is_public_network` needs to be true. Am I understanding the naming wrongly?

Reply:
Yes, 'public network' is equal to mainnet. Good thing you noticed this, as it is actually an oversight. I changed it to false now.